server/schedule/filter/filters.go

```diff
@@ -26,14 +26,37 @@ import (
 	"github.com/tikv/pd/server/core"
 	"github.com/tikv/pd/server/core/storelimit"
 	"github.com/tikv/pd/server/schedule/placement"
+	"github.com/tikv/pd/server/schedule/plan"
 	"go.uber.org/zap"
 )
 
+var (
+	okStatus                 = plan.NewStatus(plan.StatusOK)
```

Comment:
How about make `status` as the prefix? so the code editor can easily remind the developer once write down `status`